enterprise-suite/scripts/lbc.py

```diff
@@ -558,8 +611,12 @@ def setup_args(argv):
                         action='store_true')
     install.add_argument('--export-yaml', help='export resource yaml to stdout',
                         choices=['creds', 'console'])
-    install.add_argument('--reuse-resources', help='try to reuse PVCs and/or cluster roles from a previous install',
-                        action='store_true')
+
+    install.add_argument('--reuse-resources', dest='reuse_resources',
+                         help='try to reuse PVCs and/or cluster roles from a previous install', action='store_true')
+    install.add_argument('--no-reuse-resources', dest='reuse_resources', help=argparse.SUPPRESS, action='store_false')
```

Comment:
Question:  How does `help=argparse.SUPPRESS` work?

Reply:
It doesn't print a help line for that argument. We have `--reuse-resources` documented and now it says that default is `true`. To turn such flag off, standard is to have the same argument with prefix `no`.

Reply:
Got it.  Tnx.